both/common.js

```diff
@@ -1,3 +1,6 @@
 SimpleSchema.messages({
-    "passwordMismatch": "Passwords do not match"
+    "passwordMismatch": "Passwords do not match",
+    "invalidQuestionSettings": "Invalid question settings",
```

Comment:
We need to remember to internationalize these strings, so they can be translated.